unliftio/src/UnliftIO.hs

```diff
@@ -4,10 +4,12 @@ module UnliftIO
   ( module Control.Monad.IO.Unlift
   , module UnliftIO.Async
   , module UnliftIO.Chan
+  , module UnliftIO.Environment
   , module UnliftIO.Exception
   , module UnliftIO.IO
   , module UnliftIO.IORef
   , module UnliftIO.MVar
+  , module UnliftIO.Process
```

Comment:
I'm concerned with exporting both of these modules from `UnliftIO` itself, since they're likely to conflict with names that downstream packages may want to use. In particular, I can imagine people wanting to use `Text`-based values for environment handling, and `rio` is already providing a `RIO.Process` module that is based on `typed-process` instead of `process`.